### PR TITLE
Feature/header promo mailchipm api

### PR DESCRIPTION
**Pull request by recallfx**

### CODE REVIEW

File: docs/components/header.html [hunk 1]
```diff
@@ -44,21 +44,36 @@
...
-          </p>
+        <div>
+          <div class="form-submit">
+            <div class="field is-grouped">
+              <p class="control is-expanded">
+                <input class="input" name="EMAIL" type="email">
+                <label>Enter you email</label>
+              </p>
+
+              <div style="position: absolute; left: -5000px;" aria-hidden="true">
```
Comment (donatas-oberlo) [inline docs/components/header.html:59]: I suggest moving style to scss file.

File: src/js/subscribe-input.js [hunk 2]
```diff
@@ -4,42 +4,131 @@ function validateEmail(email) {
...
-    function makeInputActive() {
-      $input.parentNode.parentNode.classList.add('is-active');
-      $input.focus();
-    }
+  function makeInputActive() {
+    $input.parentNode.parentNode.classList.add('is-active');
+    $input.focus();
+  }
+
+  if ($doneHeader && $doneButton && $input && $submitButton) {
```
Comment (donatas-oberlo) [inline src/js/subscribe-input.js:35]: I would suggest to clear input after user press done. So it would be empty again.

File: src/js/footer.js [hunk 3]
```diff
@@ -1,48 +1,60 @@
 function initFooterMenu() {
   const menuTitles = document.querySelectorAll('a.footer-menu-title');
+  const $activeElement = document.querySelector('.footer-menu.is-active');
 
-  menuTitles.forEach(title => title.addEventListener('click', (event) => {
-    const activeElement = document.querySelector('.footer-menu.is-active');
-    if (activeElement) {
-      activeElement.classList.remove('is-active');
+  menuTitles.forEach($title => $title.addEventListener('click', (event) => {
```
Comment (donatas-oberlo) [inline src/js/footer.js:5]: 👍

**Review by donatas-oberlo — CHANGES_REQUESTED** [review]

File: docs/components/header.html (hunk 1 above)
Comment (recallfx) [reply docs/components/header.html:59]: This is a special case, and should not be in styles.

File: src/js/subscribe-input.js (hunk 2 above)
Comment (recallfx) [reply src/js/subscribe-input.js:35]: It is cleared in 58th line

**Review by donatas-oberlo — APPROVED** [review]

**Pull request merged**